base/abstractarray.jl

```diff
@@ -842,6 +842,12 @@ Represents the array `y` as an array having the same indices type as `x`.
 """
 of_indices(x, y) = similar(dims->y, oftype(indices(x), indices(y)))
 
+
+"""
+    full(F)
+
+Reconstruct the matrix `A` from the factorization `F=factorize(A)`.
+"""
```

Comment:
(Tangent: This docstring for `full` only captures one of the various meanings of `full`. Perhaps we should flesh out this docstring? Or not worry about it in anticipation of complete excision of full in this release cycle?)